### PR TITLE
Adds a burger made from Chunk of Blob

### PR DESCRIPTION
**Pull request by muzcl**
unknown
## About the PR
Adds a new food item called 'It Came from The Oven', made by putting two Chunks of Blob into an oven for around five seconds.

It also adds a new way to get the night vision food effect, which (forgive me if I'm wrong) only has one food that gives it.

## Why's this needed?
Chunk of Blob doesn't feel special for being a material dropped by an antagonist, so this PR alleviates it a bit by making it able to be turned into a special burger.

## Changelog

```changelog
(u)Idiot
(*)Added It Came from The Oven, a burger made out of blob.
```

### CODE REVIEW

**Comment by muzcl** [comment]
This is my first pull request, if I screwed something up let me know.

File: code/datums/cookingrecipes.dm [hunk 1]
```diff
@@ -172,6 +172,12 @@ ABSTRACT_TYPE(/datum/cookingrecipe)
 	cookbonus = 6 // still mostly raw, since we don't kill it
 	output = /obj/item/reagent_containers/food/snacks/burger/aburgination
 
+/datum/cookingrecipe/itcamefromtheoven
+	item1 = /obj/item/material_piece/wad/blob
```
Comment (Tarmunora) [inline code/datums/cookingrecipes.dm:176]: should have dough in the recipe, to match other burgs

Comment (muzcl) [reply code/datums/cookingrecipes.dm:176]: the sprite is entirely blob colors, so it'd be a bit weird having dough in the recipe but no dough in the sprite

Comment (muzcl) [reply code/datums/cookingrecipes.dm:176]: the recipe also requires two chunks of blob because one is meant to be the "dough", and the other "meat"

**Comment by Chatauscours** [comment]
The changelog seems isn't properly formated, you should put the triple backticks ``` before and after it. 
Ex:
Changelog:
``` 
(u)Abcd
(*) def
```

**Pull request closed without merging**